src/components/MenuItem/MenuDivider.module.css

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.divider {
+  margin-inline: var(--cpd-space-4x);
```

Comment:
I really like your use of logical properties! I think there's some lint rules we can add to force the project using that over the physical properties, and that would be a good step towards better RTL support.

I filed https://github.com/vector-im/compound/issues/233 for future reference

Reply:
I agree that we should enforce this! PR here: https://github.com/vector-im/compound-web/pull/101